tests/integration/provider_gov_hooks.go

```diff
@@ -59,13 +53,19 @@ func (s *CCVTestSuite) TestGetConsumerAdditionLegacyPropFromProp() {
 		Amount:      sdk.NewCoins(sdk.NewCoin("stake", math.OneInt())),
 	}
 
+	// create a legacy proposal
 	textProp, err := v1.NewLegacyContent(
 		v1beta1.NewTextProposal("a title", "a legacy text prop"),
 		authtypes.NewModuleAddress("gov").String(),
 	)
 	s.Require().NoError(err)
 
-	addConsumerProp, err := v1.NewLegacyContent(
+	// create a valid consumer addition proposal
+	addConsumerProp := testkeeper.GetTestMsgConsumerAddition()
```

Comment:
Should we call this a `msgConsumerAddition` or something similar since at this state it is not a `ConsumerAdditionProposal`?